pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
Revert. You are most likely using a pnpm version (v8.6.0 or v8.6.1) which incremented the package version causing many issues. Update your pnpm to the latest version, which no longer has this breaking change.
```suggestion
lockfileVersion: '6.0'
```

Reply:
alr i reverted it, should be good now